File: documentation-examples-kotlin/src/kotlin/dev/typr/foundationskt/docs/sqlserver/UuidType.kt
package dev.typr.foundationskt.docs.sqlserver

import dev.typr.foundationskt.*
import dev.typr.foundationskt.data.*
import java.util.UUID

@Suppress("unused")
class UuidType {
    //start
    val uuidType: SqlServerType<UUID> = SqlServerTypes.uniqueidentifier
    //stop
}

File: documentation-examples-scala/src/scala/dev/typr/foundationssc/docs/sqlserver/UuidType.scala
package dev.typr.foundationssc.docs.sqlserver
import dev.typr.foundationssc.*
import dev.typr.foundationssc.data.*


import java.util.UUID

@SuppressWarnings(Array("unused"))
object UuidType:
  //start
  val uuidType: SqlServerType[UUID] = SqlServerTypes.uniqueidentifier
  //stop
